doc/source/theory/fluid_dynamics/stabilization.rst

```diff
@@ -4,17 +4,70 @@ On the need for stabilization
 It is well known that, in the absence of stabilization, the choice of the velocity and pressure finite element spaces must be done to ensure that the `Ladyzenskaya-Babuska-Brezzi (LBB) inf-sup <https://en.wikipedia.org/wiki/Ladyzhenskaya%E2%80%93Babu%C5%A1ka%E2%80%93Brezzi_condition>`_ condition is met. It is also known that the Galerkin approximation of the Navier–Stokes equations may fail in convection dominated flows, for which there are boundary layers where the velocity solution and its gradient exhibit rapid variation over short length scales. In these regions, the classical Galerkin approach may lead to numerical oscillations which may greatly pollute the solution over the entire domain. Stabilization of the elements, which is used in Lethe, can circumvent the limitations of the classical Galerkin approach and alleviate the  need to use LBB stable elements. This notably allows for the use of equal order elements (such as P1-P1).
 
 
-SUPG/SSPG monolithic formulation
+SUPG/PSPG monolithic formulation
 -----------------------------------
-**Under construction**
 
+This approach consists of two new terms that are added to the classic weak formulation. The first one is known as SUPG (Streamline-Upwind/ Petro-Galerkin) formulation and it is in charge of stabilizing the formulation for convection-dominated flows. The second one is known as PSPG (Pressure-Stabilizing/ Petrov-Galerkin) and as its name indicates, it reduces pressure oscillations when using equal order elements. Thus, the new weak form of the Navier-Stokes equation is given by:
```

Comment:
what does "equal order elements" mean?

Reply:
It means the same order of elements for both velocity and pressure, for example Q1-Q1 or Q2-Q2 :)

Reply:
Yes, good idea. I will add something like: (e.g. Q1 for the velocity and Q1 for the pressure).